tooltester/tooltesterConfig.yml

```diff
@@ -0,0 +1,7 @@
+SERVER-URL: <ENTER SERVER URL HERE>
+TOKEN: <ENTER DOCKSTORE TOKEN HERE>
+AWS-AUTHORIZATION: <ENTER AWS PROFILE NAME HERE>
+WDL-WES-URL: <ENTER URL RETRIEVED FROM RUNNING 'agc context describe' on the WDL context HERE>ga4gh/wes/v1
+CWL-WES-URL:  <ENTER URL RETRIEVED FROM RUNNING 'agc context describe' on the CWL context HERE>ga4gh/wes/v1
+WDL-ECS-CLUSTER: <ENTER ECS Cluster name for WDL context HERE>
+CWL-ECS-CLUSTER: <ENTER ECS Cluster name for CWL context HERE>
```

Comment:
No need to change, but just noting that if there's anymore CWL and WDL fields added in the future, could format the tooltester config as an INI file instead and you can have separate CWL and WDL sections like:

```
[cwl]
wes-url: <url>
ecs-cluster: <cluster>

[wdl]
wes-url: <url>
ecs-cluster: <cluster>
```

Reply:
That is a good idea. Definitely worth considering if we do add more fields.

I think the yaml solution works alright for now.

But I definitely agree that if we add more fields we should switch to INI.